src/Intents.tsx

```diff
@@ -110,6 +110,10 @@ const intents : {[key: string]: [string[], number]} = {
   DIRECT_MESSAGE_TYPING: [[
     "TYPING_START"
   ], 14],
+    
+  MESSAGE_CONTENT: [[
+      "MESSAGE_CONTENT"
```

Comment:
this should be empty. MESSAGE_CONTENT doesnt have any events related to it https://discord.com/developers/docs/topics/gateway#list-of-intents